doc/source/contributing/developer_guide.md

```diff
@@ -2,54 +2,53 @@
 
 # Developer Guide
 ## Library architecture
-ArviZ is organized in modules (the folders in https://github.com/arviz-devs/arviz/tree/main/arviz).
-The main 3 modules are `data`, `plots` and `stats`,
-and then the `tests` folder is where tests for all these 3 modules are.
+ArviZ is organized in modules (the folders in [arviz directory](https://github.com/arviz-devs/arviz/tree/main/arviz)).
+The main 3 modules are `data`, `plots` and `stats`.
+The [tests](https://github.com/arviz-devs/arviz/tree/main/arviz/tests) folder contains tests for all these 3 modules.
 
-The `static` is only used to store style and css files to get html output for `InferenceData` and
-`wrappers` is still experimental (not tested yet either) and interacts closely with
-both `data` and `stats` modules.
+[static](https://github.com/arviz-devs/arviz/tree/main/arviz/static) is only used to store style and CSS files to get HTML output for `InferenceData`.
+[wrappers](https://github.com/arviz-devs/arviz/tree/main/arviz/wrappers) is still experimental (not tested yet either) and interacts closely with both [data](https://github.com/arviz-devs/arviz/tree/main/arviz/data) and [stats](https://github.com/arviz-devs/arviz/tree/main/arviz/stats) modules.
```

Comment:
```suggestion
The [static](https://github.com/arviz-devs/arviz/tree/main/arviz/static) folder is only used to store style and CSS files to get HTML output for `InferenceData`. Finally we have the [wrappers](https://github.com/arviz-devs/arviz/tree/main/arviz/wrappers) folder. Which contains experimental (not tested yet either) features and interacts closely with both [data](https://github.com/arviz-devs/arviz/tree/main/arviz/data) and [stats](https://github.com/arviz-devs/arviz/tree/main/arviz/stats) modules.
```

Reply:
This and the previous suggestion should be part of the same paragraph. (github did not allowed me to create a single suggestion)